samples/bluetooth/hci_vs_scan_req/README.rst

```diff
@@ -0,0 +1,30 @@
+.. _bluetooth-hci-vs-scan-req-sample:
+
+Bluetooth: HCI VS Scan Request
+##############################
+
+Overview
+********
+
+This simple application is a usage example to manage HCI VS commands to obtain
+scan equest events even using legacy advertisements, while may result in lower
+RAM usage than using extended advertising.
+This is quite important in applications in which the broadcaster role is added
+to the central role, where the RAM saving can be bigger.
+This sample implements only the broadcaster role; the peripheral role with
```

Comment:
A last comment and then I'll approve:
The sample can be slimmed down even more, by removing the `CONFIG_BT_PERIPHERAL` support.

I don't see it adding anything related to this VS event in the code. Users can easily copy code from peripheral samples if they need it.

Reply:
I agree with this. Since the purpose of this sample is to show the VS command, then it should do just that.

Reply:
removed